### PR TITLE
[MRG] Adds FAQ regarding initialization behavior of NeuralNet

### PR DESCRIPTION
**Pull request by thomasjpfan**
Puts the information in https://github.com/dnouri/skorch/issues/376#issuecomment-433732676 into the FAQ.

### CODE REVIEW

File: docs/user/FAQ.rst [hunk 1]
```diff
@@ -150,3 +150,49 @@ it to :func:`~skorch.helper.predefined_split`:
     )
 
     net.fit(X_train, y_train)
+
+
+What happens when NeuralNet is passed a initialized Pytorch module?
```
Comment (BenjaminBossan) [inline docs/user/FAQ.rst]:
```suggestion
What happens when NeuralNet is passed an initialized Pytorch module?
```

**Comment by BenjaminBossan** [comment]
Apart from the typo, this LGTM

**Comment by thomasjpfan** [comment]
Error coming from https://github.com/box/flaky/pull/140. There seems to be some backward incompatible changes in pytest 4.1.

**Comment by BenjaminBossan** [comment]
There has been a new flaky release after the PR, perhaps retriggering solves the issue.

**Comment by thomasjpfan** [comment]
Everything looks good now! :)

**Comment by BenjaminBossan** [comment]
Good work, thanks.

**Pull request merged**